docs/get_started/get_started_demos.md

```diff
@@ -445,42 +444,25 @@ The following commands run the Image Classification Code Sample using the [dog.b
 
 @endsphinxdirective
 
-When the sample application is complete, you see the label and confidence for the top 10 categories on the display. Below is a sample output with inference results on CPU:
 
-@sphinxdirective
-
-   .. code-block:: sh
-
-   Top 10 results:
+## Other Demos and Samples
 
-   Image dog.bmp
-
-      classid probability label
-      ------- ----------- -----
-      156     0.6875963   Blenheim spaniel
-      215     0.0868125   Brittany spaniel
-      218     0.0784114   Welsh springer spaniel
-      212     0.0597296   English setter
-      217     0.0212105   English springer, English springer spaniel
-      219     0.0194193   cocker spaniel, English cocker spaniel, cocker
-      247     0.0086272   Saint Bernard, St Bernard
-      157     0.0058511   papillon
-      216     0.0057589   clumber, clumber spaniel
-      154     0.0052615   Pekinese, Pekingese, Peke
+See the [Samples](../OV_Runtime_UG/Samples_Overview.md) page for more sample applications. Each sample page explains how the application works and shows how to run it. Use the samples as a starting point that can be adapted for your own application.
 
-@endsphinxdirective
+OpenVINO also provides demo applications for using off-the-shelf models from [Open Model Zoo](@ref model_zoo). These give even more examples of how to run model inference with the OpenVINO API. To run the demos in Open Model Zoo, you need to build OpenVINO demos first:
```

Comment:
Again, I would probably remove the tabs and most of the following content, leaving just the general info and referring readers to the OMZ Demos page.

Reply:
Agreed, I had only left the OMZ Demos content in there because it was part of the original article. I removed the section linking to OMZ Demos build instructions, and put this instead:

"OpenVINO also provides demo applications for using off-the-shelf models from [Open Model Zoo](@ref model_zoo). Visit [Open Model Zoo Demos](@ref omz_demos) if you'd like to see even more examples of how to run model inference with the OpenVINO API."